rdf-delta-integration-tests/src/test/java/org/seaborne/delta/AbstractTestDeltaLink.java

```diff
@@ -620,9 +620,20 @@ public void dLink_patch_events_graph_1() {
         // Agrees with "patch1"
         Graph g = GraphFactory.createGraphMem();
         AtomicInteger counter = new AtomicInteger();
-        GraphListener gl = new GraphListenerBase() {
-            @Override protected void deleteEvent(Triple t) {}
-            @Override protected void addEvent(Triple t) { counter.incrementAndGet(); }
+        GraphListener gl = new GraphListener() {
+            @Override public void notifyAddArray(Graph g, Triple[] triples) { for (Triple triple : triples) { addEvent(triple); } }
+            @Override public void notifyAddGraph(Graph g, Graph added) {}
+            @Override public void notifyAddIterator(Graph g, Iterator<Triple> it) { for ( ; it.hasNext(); ) addEvent(it.next()); }
+            @Override public void notifyAddList(Graph g, List<Triple> triples) { notifyAddIterator(g, triples.iterator()); }
+            @Override public void notifyAddTriple(Graph g, Triple t) { addEvent(t); }
+            @Override public void notifyDeleteArray(Graph g, Triple[] triples) {}
+            @Override public void notifyDeleteGraph(Graph g, Graph removed) {}
+            @Override public void notifyDeleteIterator(Graph g, Iterator<Triple> it) {}
+            @Override public void notifyDeleteList (Graph g, List<Triple> triples) {}
+            @Override public void notifyDeleteTriple(Graph g, Triple t) {}
+            @Override public void notifyEvent(Graph source, Object value) {}
+            protected void deleteEvent(Triple t) {}
+            protected void addEvent(Triple t) { counter.incrementAndGet(); }
```

Comment:
Looks like `deleteEvent` can be deleted and `addEvent` should be private

Reply:
Done